.github/workflows/team-slack-bot.yml

```diff
@@ -1,17 +0,0 @@
-name: slack-notifications-prs-issues
-
-on:
-  pull_request:
-    types: [ready_for_review]
-  issues:
-    types: [opened]
-
-jobs:
-  slack-action:
-    runs-on: ubuntu-latest
-    name: Notify Team in Slack
-
-    steps:
-      - name: Curl to webhook
-        run: |
-          curl -X POST -H 'Content-type: application/json' --data '{"text":"A new ${{ github.event_name }} has been published. Please review ${{ github.event.pull_request.html_url }}${{ github.event.issue.html_url }}"}' ${{ secrets.SLACK_WEBHOOK }}
```

Comment:
Will you remove also the SLACK_WEBHOOK from secrets please?

Reply:
yeah. i never checked the slack channel and it was just annoying. Users can subscribe to email notifications if they care about this kind of stuff anyhow.

Reply:
Yes, I agree, I don't even know which channel it was :D